VocaDbWeb/Views/Help/Index.cshtml

```diff
@@ -131,7 +134,7 @@
 		<br />
 		<h4 id="contactus">Contact Us</h4>
 		support[at]vocadb.net<br />
-		<a href="http://www.vocaloidotaku.net/index.php?/topic/25685-vocadb-vocaloid-database" title="">Discussion thread at VocaloidOtaku</a><br />
+		//<a href="http://www.vocaloidotaku.net/index.php?/topic/25685-vocadb-vocaloid-database" title="">Discussion thread at VocaloidOtaku</a><br />

```

Comment:
Sorry, but comments like these don't work in HTML. You'll have to wrap them in HTML comment tags, see https://html.com/tags/comment-tag/

About this particular line, I think it could be just removed too, it can always be restored from source control if necessary.

Reply:
Ah, thanks! I tried the double slashes and saw the line become "grayed-out" so I just assumed it worked.
I've deleted some comments and changed the others!